packages/contracts/test/deploy/managing-dao.ts

```diff
@@ -17,6 +17,11 @@ import {
 } from '../../typechain';
 
 import daoArtifactData from '../../artifacts/src/core/dao/DAO.sol/DAO.json';
+import daoRegistryArtifactData from '../../artifacts/@aragon/osx-v1.0.1/framework/dao/DAORegistry.sol/DAORegistry.json';
```

Comment:
Disclaimer: after adding the contracts to the migrations file, the tests in these files were complaining because of ambiguity about which artifact to choose.
Accordingly, I had to add the specific artifact data.